### ISSUE TITLE
ignore: need to ensure a linebreak

### ISSUE THREAD

**Issue by jankatins** [tj/git-extras#476]
If `.gitignore` does not include a linebreak (because I forgot...), the `git ignore xxx/` will include the new ignore pattern at the end _without_ a leinreak inbetween:

This `.gitignore`:

```
pattern/
# no linebreak at the end of the file
```

```
git ignore build/
```

will result in:

```
pattern/build/
# linbreak
```

**Comment by tfendin** [comment]
This was fixed in #949. Could be closed

**Comment by spacewander** [comment]
Thanks for @tfendin!
